### PR TITLE
Reply button for thread view

### LINKED ISSUE #936: Reply button is missing on message screen
Simulator Screen Shot - iPhone 12 - 2021-11-05 at 22 18 06

### LINKED ISSUE #940: expand/close icon is white on white background
Minor UI issue but could confuse user:

Simulator Screen Shot - iPhone 11 Pro - 2021-11-06 at 20 28 57

with red color:

Simulator Screen Shot - iPhone 11 Pro - 2021-11-06 at 20 30 52

### PR DESCRIPTION
**Pull request by sosnovsky**
This PR adds reply button to thread view and fixes expand/close button color.

close #936 
close #940

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests will be added later - #937

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift [hunk 1]
```diff
@@ -90,24 +90,23 @@ extension ThreadDetailsViewController {
     private func expandThreadMessage() {
         let indexOfSectionToExpand = thread.messages.firstIndex(where: { $0.isMessageRead == false }) ?? input.count - 1
         let indexPath = IndexPath(row: 0, section: indexOfSectionToExpand + 1)
-        handleTap(at: indexPath)
+        handleExpandTap(at: indexPath)
     }
 
-    private func handleTap(at indexPath: IndexPath) {
-        guard let threadNode = node.nodeForRow(at: indexPath) as? TextImageNode else {
+    private func handleExpandTap(at indexPath: IndexPath) {
```
Comment (tomholub) [inline FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift:96]: This name is much better - thank you!

**Review by tomholub — COMMENTED** [review]
Code looks good. Thanks! Will test.

**Review by tomholub — APPROVED** [review]
Works well. Maybe we should move the reply icon a tiny bit further from the chevron icon? Seem like they are a bit too close together. For spacing, can follow the space between the header icons which are comfortably apart. (people with fat fingers etc)

**Comment by sosnovsky** [comment]
> Works well. Maybe we should move the reply icon a tiny bit further from the chevron icon? Seem like they are a bit too close together. For spacing, can follow the space between the header icons which are comfortably apart. (people with fat fingers etc)

Agree, looks quite close. I was also thinking about removing `expand` icon when message is expanded, so it'll be only `reply` button, as currently `expand` icon looks too repetitive, as for me. What do you think?

**Pull request merged**